src/nvofapi/nvofapi_vk_instance.cpp

```diff
@@ -65,6 +65,12 @@ namespace dxvk {
         VK_GET_DEVICE_PROC_ADDR(vkBindOpticalFlowSessionImageNV);
         VK_GET_DEVICE_PROC_ADDR(vkCmdOpticalFlowExecuteNV);
 
+        // Confirm that OPTICAL_FLOW extension is available
+        if (!m_vkCreateOpticalFlowSessionNV || !m_vkDestroyOpticalFlowSessionNV || !m_vkBindOpticalFlowSessionImageNV || m_vkCmdOpticalFlowExecuteNV) {
```

Comment:
You missed the last `!`:

```suggestion
        if (!m_vkCreateOpticalFlowSessionNV || !m_vkDestroyOpticalFlowSessionNV || !m_vkBindOpticalFlowSessionImageNV || !m_vkCmdOpticalFlowExecuteNV) {
```

Reply:
I hope nobody saw that ;)